test/ConverterTest.php

```diff
@@ -4,6 +4,22 @@
 
 class ConverterTest extends \PHPUnit_Framework_TestCase
 {
+    public function testRemovesColorProfileInformation()
+    {
+        $bin = self::converter()->convert(file_get_contents(__DIR__ . '/data/colorProfile.jpeg'), self::command('strip'));
+        $tmpFile = self::tmpDir() . 'profilesCheckRemove.jpg';
+        file_put_contents($tmpFile, $bin);
```

Comment:
Why you don't delete temp files after usage?

Reply:
fixed in #13